dags/map_reproducibility/aotc_reproducibility.py

```diff
@@ -0,0 +1,96 @@
+"Bash helper commands for AOTC artifacts"
```

Comment:
Could you add Copyright header, same as another file? Thanks! 

```
# Copyright 2024 Google LLC
....
```

Reply:
done